server/test/write.go

```diff
@@ -28,7 +28,7 @@ type writeCommandTest struct {
 var tk = &openfgapb.TupleKey{
```

Comment:
Refactor this to `tuple.NewTupleKey("repository:openfga/openfga", "administrator", "user:github|alice@openfga")`